src/routes/+page.server.ts

```diff
@@ -26,11 +27,19 @@ export const actions: Actions = {
       steps,
       technical
     }) => {
-      const priorityLabel: Record<string, string> = {
-        low: '🟢  **Low**',
-        medium: '🟡  **Medium**',
-        high: '🟠  **High**',
-        critical: '🔴  **Critical**'
+      const getPriorityLabel = (priority: string) => {
+        if (priority === priorityType.Low) {
+          return '🟢  **Low**';
+        }
+        if (priority === priorityType.Medium) {
+          return '🟡  **Medium**';
+        }
+        if (priority === priorityType.High) {
+          return '🟠  **High**';
+        }
+        if (priority === priorityType.Critical) {
+          return '🔴  **Critical**';
+        }
```

Comment:
Also, do you agree to return a default value at the end?

like this:
```suggestion
      const getPriorityLabel = (priority: string) => {
        if (priority === priorityType.Medium) {
          return '🟡  **Medium**';
        }
        if (priority === priorityType.High) {
          return '🟠  **High**';
        }
        if (priority === priorityType.Critical) {
          return '🔴  **Critical**';
        }
        return '🟢  **Low**';
```

Basically it's all the same, but you guarantee that a string is always returned. I think it's more safe